README.md

```diff
@@ -1,7 +1,8 @@
 # webdev
```

Comment:
I tried to replace with with something more catchy that could also be a hashtag on Twitter. My ideas:
- browsercompat
- compatbug
- compatissue
- webcompat (except that's taken)